lighthouse-core/report/v2/renderer/details-renderer.js

```diff
@@ -81,6 +83,36 @@ class DetailsRenderer {
     element.appendChild(items);
     return element;
   }
+
+  /**
+   * @param {!CardsDetailsJSON} details
```

Comment:
`render()` needs to take a `(DetailsRenderer.DetailsJSON|DetailsRenderer.CardsDetailsJSON)` now...this is why we need to turn type checking back on :)